version: "1.0.0"

safety_timeout: 1.0 # [s]

main_timer_rate: 30 # [Hz]

topic_check:

  enabled: true
  timeout: 5.0 # [s], a window for the last accepted message

  # if "/" is provided at the beginning of topic_name, the topic name is considered as written,
  # if there is no "/" at the beginning, UAV_NAME is automatically added in from of the topic name (/uav1/topic_name)
  topics: [
    "odometry/uav_state",
    "control_manager/diagnostics",
    "uav_manager/diagnostics",
    "mavros/local_position/odom",
  ]
